panther_analysis_tool/main.py

```diff
@@ -357,6 +372,61 @@ def update_schemas(args: argparse.Namespace) -> Tuple[int, str]:
     return 0, ""
 
 
+def generate_release_assets(args: argparse.Namespace) -> Tuple[int, str]:
+    # First, generate the appropriate zip file
+    # set the output file to appropriate name for the release: panther-analysis-all.zip
+    release_file = args.out + "/" + "panther-analysis-all.zip"
+    signature_filename = args.out + "/" + "panther-analysis-all.sig"
+    return_code, archive = zip_analysis(args)
```

Comment:
Did we decide if we also want to create the `panther-analysis-policies.zip` and `panther-analysis-rules.zip`? No strong preference from me either way.

Reply:
Here are some stats on download counts for the various files for a few of the most recent releases - I would vote on not including the -policies and -rules files. 

```
Release: v1.16.0: Added packs directory
	panther-analysis-all.sig: 199
	panther-analysis-all.zip: 246
Release: v1.15.0: Added DataModels and Standard Rules
	managed-schemas.zip: 5
	panther-analysis-all.zip: 73
	panther-analysis-policies.zip: 1
	panther-analysis-rules.zip: 1
Release: v1.14.0: Added support for IOCs
	panther-analysis-all.zip: 316
	panther-analysis-iocs.zip: 5
	panther-analysis-policies.zip: 3
	panther-analysis-rules.zip: 4
Release:
	panther-analysis-all.zip: 62
Release:
	panther-analysis-all.zip: 108
	panther-analysis-policies.zip: 5
	panther-analysis-rules.zip: 11
Release:
	panther-analysis-all.zip: 171
	panther-analysis-policies.zip: 6
	panther-analysis-rules.zip: 11
Release:
	panther-analysis-all.zip: 169
	panther-analysis-policies.zip: 8
	panther-analysis-rules.zip: 8
Release:
	panther-analysis-all.zip: 83
	panther-analysis-policies.zip: 7
	panther-analysis-rules.zip: 6
Release:
	panther-analysis-all.zip: 132
	panther-analysis-policies.zip: 7
	panther-analysis-rules.zip: 7
```